kubernetes/cloud-config.yml

```diff
@@ -0,0 +1,22 @@
+azs:
+- name: z1
+
+vm_types:
+- name: default
+
+disk_types:
+- name: default
+  disk_size: 1024
+
+networks:
+- name: default
+  type: dynamic
+  subnets:
+  - azs: [z1]
```

Comment:
What are you missing specifically? 
/etc/resolve.conf looks like the following on Pods/Containers created by the CPI:
```
nameserver 10.0.0.10
search default.svc.cluster.local svc.cluster.local cluster.local
options ndots:5
```